src/components/SearchBar.tsx

```diff
@@ -23,7 +23,7 @@ export default function SearchBar() {
   });
 
   const onSelectionChange = (key: Key | null) => {
-    const selectedTitle = (titles.items as SimpleTitle[]).find(title => title.id === key)?.name;
+    const selectedTitle = (titles.items).find(title => title.id === key)?.name;
```

Comment:
```suggestion
    const selectedTitle = titles.items.find(title => title.id === key)?.name;
```
Trenger ikke parantesen heller lengre med fiksen din!